### DISCUSSION TITLE
Raspberryshake ile proje geliştirebilir miyiz?

### DISCUSSION THREAD

**Discussion by trhero** [acikkaynak/deprem-yardim-projesi#76] [Ideas]
ilk başta cihazların Türkiye'de arttırılması için bir kampanya şuan Türkiye'de sadece 1 cihaz var.
https://stationview.raspberryshake.org/#/?lat=42.93674&lon=30.72130&zoom=5.000&net=AM&sta=RF9F7
cihaz link : https://shop.raspberryshake.org/product/turnkey-iot-home-earth-monitor-rs-3d/?attribute_pa_variation=indoor&attribute_pa_license=private-use-125-discount

Cihazların artmasıyla yer bilimci arkadaşlarla beraber çalışılarak  erken uyarı sistemi ve AI destekli deprem tahmin  yazılımları geliştire bilinir. 
Yeterli veriyle daha fazla projede geliştirilebileceğini düşünüyorum.

**Comment by sercangoger** [comment]
çok teşekkür ederim herkes adın fikriniz için. 

cihaz hakkında detaylı bilgi verirmisiniz. ekipler bu konu da daha fazla bilgi alarak neler yapılabilir düşünebilmeleri için.
bu cihazı almak mı doğru. yoksa kart dizayn arkadaşları bu cihazı daha az maliyet ile tassrlayabilirler mi .
yoksa almak mı doğru.
bir çok şey yapılabilir.

**Reply by trhero** [reply]
Merhaba teşekkürler öncelikle,
Kesinlikle yapabiliriz ama daha uyguna mı olur daha pahalımı bu konuda daha bilgisi olan bir arkadaş yardımcı olursa sevinirim.
Ana mantık çok basit yerin haraketini izleyen Jeofon Sensörü ve sensörden gelen bilgiyi derleyip yayın yapan Raspberry Pi kullanılıyor.
Daha uyguna libre, orange pi, Arduino kullanıla bilinir jeafon Sensörleri daha uygun muadil buluna bilinir. Çok fazla donanım bilgim olmadığı için kart tasarlanır mı emin değilim.

Raspberryshake kullanılan Jeofon Sensörü:
https://raspberryshake.org/wp-content/uploads/2020/05/racotech-geophone-datasheet.pdf
Burda da cihazın temal tasarımı anlatılmış
https://raspberryshake.org/about/technology/

**Reply by sercangoger** [reply]
çok güzel burada ki yetkili arkadaşlar konuuyu görürlerse eğer değerlendirirler umarım.
